dynawo/sources/Modeler/Common/DYNModelMulti.cpp

```diff
@@ -1150,13 +1150,18 @@ void ModelMulti::printVariableNames() {
   }
   nVar = 0;
   Trace::debug(Trace::variables()) << "------------------------------" << Trace::endline;
-  Trace::debug(Trace::variables()) << "X variables" << Trace::endline;
+  Trace::debug(Trace::variables()) << "X variables (with initial type)" << Trace::endline;
```

Comment:
We should adapt the header depending on withVariableType

Reply:
done